app/data/15.3.yml.erb

```diff
@@ -0,0 +1,203 @@
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-KDE-Live-x86_64-Media.iso.meta4
+      - name: <%= _("Pick Mirror") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-KDE-Live-x86_64-Media.iso?mirrorlist
+      - name: <%= _("Checksum") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-KDE-Live-x86_64-Media.iso.sha256
+    - name: Rescue LiveCD
+      primary_link: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-x86_64-Media.iso
+      links:
+      - name: <%= _("Metalink") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-x86_64-Media.iso.meta4
+      - name: <%= _("Pick Mirror") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-x86_64-Media.iso?mirrorlist
+      - name: <%= _("Checksum") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-x86_64-Media.iso.sha256
+  - name: aarch64
+    types:
+    - name: Gnome
+      primary_link: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-GNOME-Live-aarch64-Media.iso
+      links:
+      - name: <%= _("Metalink") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-GNOME-Live-aarch64-Media.iso.meta4
+      - name: <%= _("Pick Mirror") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-GNOME-Live-aarch64-Media.iso?mirrorlist
+      - name: <%= _("Checksum") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-GNOME-Live-aarch64-Media.iso.sha256
+    - name: KDE
+      primary_link: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-KDE-Live-aarch64-Media.iso
+      links:
+      - name: <%= _("Metalink") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-KDE-Live-aarch64-Media.iso.meta4
+      - name: <%= _("Pick Mirror") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-KDE-Live-aarch64-Media.iso?mirrorlist
+      - name: <%= _("Checksum") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-KDE-Live-aarch64-Media.iso.sha256
+    - name: Rescue LiveCD
+      primary_link: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-aarch64-Media.iso
+      links:
+      - name: <%= _("Metalink") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-aarch64-Media.iso.meta4
+      - name: <%= _("Pick Mirror") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-aarch64-Media.iso?mirrorlist
+      - name: <%= _("Checksum") %>
+        url: /distribution/leap/<%= @version %>/live/openSUSE-Leap-<%= @version %>-Rescue-CD-aarch64-Media.iso.sha256
+- name: <%= _("Ports") %>
```

Comment:
Based on the conclusions in https://en.opensuse.org/openSUSE:OpenSUSE_Distribution_Architectures_Policy I suggest to rename this to "Additional Architectures".

Reply:
I would suggest to move all archs to `installation` tab instead as they are all supported in SLE and all are tested in openQA (except s390, but this should be fixed "soon", at least before the release).

Reply:
I support the idea of keeping all install media from openSUSE:Leap:15.3 in the Installation tab. It mirrors well the project structure and simplifies UI for user. Let me update the request and update screenshot

Reply:
Done so. Moved all to installation. I had to change links a bit. All DVD, NETINSTALL, sha and meta4 links where tested locally.

![leap_installation_tab_all_arches](https://user-images.githubusercontent.com/510119/102085880-831fc780-3e17-11eb-9cdf-ba4e882abcbf.png)